src/directory/directory.mjs

```diff
@@ -866,7 +866,7 @@ export const directory = {
                 },
                 {path: 'src/pages/[platform]/tools/cli/commands.tsx'},
                 {
-                  path: 'src/pages/[platform]/build-a-backend/graphqlapi/index.mdx',
+                  path: 'src/pages/[platform]/tools/cli/graphqlapi/index.mdx',
```

Comment:
@heatheramz switching this to point to the cli graphqlapi page, if you add two entries with the same route to directories you get "interesting" behavior.  Was this change to have the cli graphqlapi section point to the build a backend section intentional?  What is the desired behavior here?

Reply:
This may have been done by Rene to get them to that location since we moved a lot of CLI docs to the build a backend b/c they were more guide material